StudentsDB/target/classes/META-INF/maven/studentsDB/StudentsDB/pom.properties

```diff
@@ -0,0 +1,7 @@
+#Generated by Maven Integration for Eclipse
+#Wed Jun 13 14:03:26 EEST 2018
+version=0.0.1-SNAPSHOT
+groupId=studentsDB
+m2e.projectName=StudentsDB
+m2e.projectLocation=C\:\\Users\\stefan.yonkov\\eclipse-workspace\\Students\\StudentsDB
```

Comment:
By the way, do we need such pom.properties file? Since there is some meta information in this file, especially considering your local PC data, so I believe we should gitignore all pom.properties files. ;)